api/swagger.yaml

```diff
@@ -7192,22 +7215,7 @@ paths:
         200:
           description: "The container has exit."
           schema:
-            type: "object"
-            title: "ContainerWaitResponse"
-            description: "OK response to ContainerWait operation"
-            required: [StatusCode]
-            properties:
-              StatusCode:
-                description: "Exit code of the container"
-                type: "integer"
-                x-nullable: false
-              Error:
-                description: "container waiting error, if any"
-                type: "object"
-                properties:
-                  Message:
-                    description: "Details of an error"
-                    type: "string"
+            $ref: "#/definitions/ContainerWaitResponse"
```

Comment:
Ahhhhh, this commit mentions the CI issue -- is there a way we could somehow get the Swagger reproducibility fix backported to the version we use instead?

Reply:
The tricky bit is that effectively "we're using it wrong"; these types are generated for endpoints, and the swagger generator is meant to produce a full client for that (client, code handling the endpoint, as well as validation and models), and we are only using it to generate some of the types / structs.

To be honest, I think we should consider to do the _reverse_; generate the swagger from our Go structs (as we're generally better at defining neat Go structs than producing YAML), also, we're constantly trying to adjust the Swagger to make it "do the right thing" (and produce Go structs the way we want them to be).

Reply:
The swagger "generated code" is also a bit awkward, in that it dumps everything in a single "types" package, which is an ill fit for Go conventions, and also, well, makes it a large bunch of (sometimes very unrelated) structs.

That said; moving the definition to the "definitions" inside the swagger may be slightly neater than "where did this weirdly named struct come from?", and (possibly) would allow some of those structs to _actually_ be reused (although, again, we may need to juggle things to make swagger do what we want 😞)

In short; I like swagger for it to generate nice looking docs, but I hate the generated bits (it's pretty poorly defined Go).